xml/admin_monitoring_alerting.xml

```diff
@@ -1385,79 +1381,79 @@ monitoring:
           <para>One or more OSDs that use the &bluestore; backend have been
             allocated db partitions (storage space for metadata, normally on
             a faster device) but that space has filled, such that metadata has
-            “spilled over” onto the normal slow device. This is not
+            overflowed onto the normal slow device. This is not
             necessarily an error condition or even unexpected, but if the
             administrator’s expectation was that all metadata would fit on
             the faster device, it indicates that not enough space was provided.
-            This warning can be disabled on all OSDs with:</para>
+            This warning can be disabled on all OSDs with the following command:</para>
 <screen>&prompt.cephuser;ceph config set osd bluestore_warn_on_bluefs_spillover false</screen>
-          <para>Alternatively, it can be disabled on a specific OSD with:</para>
+          <para>Alternatively, it can be disabled on a specific OSD with the following command:</para>
 <screen>&prompt.cephuser;ceph config set osd.123 bluestore_warn_on_bluefs_spillover false</screen>
-          <para>To provide more metadata space, the OSD in question could be
-            destroyed and reprovisioned. This will involve data migration and recovery.
-            It may also be possible to expand the LVM logical volume backing the db
+          <para>To provide more metadata space, the OSD in question can be
+            destroyed and reprovisioned. This involves data migration and recovery.
+            It is possible to expand the LVM logical volume backing the db
             storage. If the underlying LV has been expanded, the OSD daemon needs
-            to be stopped and BlueFS informed of the device size change with:</para>
```

Comment:
(Not sure this is the 1st occurrence)

If we talk about "BlueFS" a lot, should we have an entity for it?

Reply:
Can consider. Why not.